backend/README.md

```diff
@@ -35,8 +57,70 @@ Both `dev-env/run-docker-compose.sh` and `dev-env/run-db-and-local-django.sh` de
 
 * Then go to https://console.cloud.google.com/apis/credentials/consent, and add a user test (typicaly your gmail account)
 
-# Testing
+## Testing
 
 In order to ease your testing and debug time, use pytest : `pytest`
 Moreover, you can run the following command to have a complete recap in a html document for each test:
 `pytest --html=report.html --self-contained-html`
+
+## Code Quality
+
+We use several tools to keep the code quality as good, readable and maintainable
+as possible:
+- `isort` automatically sorts import statements
+- `pylint` static code analyzer which looks for errors
+- `flake8` a wrapper around three popular tools for style enforcement
+
+All of them should be automatically triggered by the continuous
+integration system.
+
+You can run them locally before creating a new commit, by running the
+following commands.
+
+**manual installation**
+
+The commands should be run from the root of the Git repository. 
+
+```shell
+# check file by file
+isort file1.py file2.py
```

Comment:
`isort` can also use a config file ".isort.cfg`

I think that something like this config would fit our needs:
```cfg
[settings]
skip = migrations
profile = black
line_length = 99
```

Reply:
Sure, I'll add this configuration in the next PR when I'll refactor all import statements.